desktop.patch

```diff
@@ -0,0 +1,9 @@
+--- old/src/re.rizin.cutter.desktop	2022-10-16 19:53:35.914579013 +0200
++++ new/src/re.rizin.cutter.desktop	2022-10-16 20:38:12.519561654 +0200
```

Comment:
instead of a patch you can use `desktop-file-edit` in the post-install.

Reply:
Thanks I did not know about this command.
This is fixed in https://github.com/flathub/flathub/pull/2251/commits/4249cea1c7be1c62c1847c2d3c773161079f9da1